.env.example

```diff
@@ -7,6 +7,9 @@ CONTAINER_IMAGE=/netscratch/enroot/nvcr.io_nvidia_pytorch_23.07-py3.sqsh
 # chooses a Slurm partition
 PARTITION=RTXA6000-SLT
 
+# set the max time limit for the job
```

Comment:
describe the format of the time limit. What does the example value of `3-0` mean?

Reply:
got it, ill add a short description on how the time format is set in slurm

Reply:
thanks!